model/project_ref.go

```diff
@@ -1787,10 +1787,15 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 			ProjectRefDisabledStatsCacheKey:      p.DisabledStatsCache,
 			ProjectRefFilesIgnoredFromCacheKey:   p.FilesIgnoredFromCache,
 		}
-		if !isRepo && !p.UseRepoSettings() {
+		if !isRepo && !p.UseRepoSettings() && !defaultToRepo {
```

Comment:
If !p.UseRepoSettings then !defaultToRepo is true right, since we aren't using repo settings anyway?

Reply:
good, point, they won't have access to the button until they attach to a repo.

Reply:
(done)